src/components/mdInputContainer/mdInputContainer.vue

```diff
@@ -7,6 +7,10 @@
     <md-button class="md-icon-button md-toggle-password" @click.native="togglePasswordType" v-if="mdHasPassword">
       <md-icon>{{ showPassword ? 'visibility_off' : 'visibility' }}</md-icon>
     </md-button>
+
+    <md-button class="md-icon-button md-clear-input" @click.native="clearInput" v-if="mdClearable && hasValue">
```

Comment:
I think the button should be highlighted with the primary color when the input receives focus. An easy way to do that is to toggle the 'md-primary' class when `this.isFocused` is true.